src/pe_reports/report_generator.py

```diff
@@ -0,0 +1,42 @@
+"""ciagov/pe-reports: A tool for creating Posture & Exposure reports."""
```

Comment:
Fix a typo by eliminating an unnecessary bit of info:
```suggestion
"""A tool for creating Posture & Exposure reports."""
```

Reply:
Resolved in 8806233